ethexe/runtime/src/wasm/api/run.rs

```diff
@@ -47,22 +45,15 @@ pub fn run(
 
     let program_state = ri.storage().read_state(state_root).unwrap();
 
-    let (journal, origin) = process_next_message(
+    let journals = process_queue(
         program_id,
         program_state,
         maybe_instrumented_code,
         original_code_id,
         &ri,
     );
 
-    log::debug!(
-        "Done creating journal: {} notes, origin {origin:?}",
-        journal.len()
-    );
-
-    for note in &journal {
-        log::debug!("{note:?}");
-    }
+    log::debug!("Done creating journal: {} notes", journals.len());
```

Comment:
its not notes, its journals, so I'd keep logging everything

Reply:
Fixed